crates/api_common/src/utils.rs

```diff
@@ -986,6 +987,16 @@ fn build_proxied_image_url(
   ))
 }
 
+pub fn check_comment_depth(comment: &Comment) -> LemmyResult<()> {
+  let path = &comment.path.0;
+  let length = path.split('.').count();
+  if length > MAX_COMMENT_DEPTH_LIMIT {
```

Comment:
I'm curious as to why this is `>` instead of `>=`. Based on this PR and places where it's currently used ([usage 1](https://github.com/LemmyNet/lemmy/blob/8cf14f407fba600b2f09c8fb37aa08719a4b1295/crates/api/api_crud/src/comment/create.rs#L92), [usage 2](https://github.com/LemmyNet/lemmy/blob/8cf14f407fba600b2f09c8fb37aa08719a4b1295/crates/apub_objects/src/objects/comment.rs#L200)) this is used to check the depth of the parent of the comment being replied to. However, since the parent is a comment that already exists, wouldn't it always be below the max depth limit since it wouldn't have been able to be posted otherwise?

Reply:
Fixed, thanks.